ios/SplunkOtelReactNative.swift

```diff
@@ -59,6 +59,13 @@ class SplunkOtelReactNative: NSObject {
         resolve(spanExporter.export(spans: spans))
     }
     
+    @objc(nativeCrash)
+    func nativeCrash() -> Void {
```

Comment:
This works fine without the resolve blocks? :t-rex:

Reply:
Yes